routers/web/user/home.go

```diff
@@ -448,8 +448,8 @@ func buildIssueOverview(ctx *context.Context, unitType unit.Type) {
 	// - Team org's owns the repository.
 	// - Team has read permission to repository.
 	repoOpts := &repo_model.SearchRepoOptions{
-		Actor:       ctx.Doer,
-		OwnerID:     ctx.Doer.ID,
+		Actor:       ctxUser,
```

Comment:
Why `Actor` should be `ctxUser`? Maybe it should still be `ctx.Doer`?

Reply:
Oops, good question.

Reply:
You are right, Actor should be `ctx.Doer`. 
Copied from Milestones render function. We should also fix it. 😕

Reply:
Done in https://github.com/go-gitea/gitea/pull/26695/commits/f5dd6b5072551c6001e26f7169ee756ad6de4cfc